elementary/monitor/api/groups/groups.py

```diff
@@ -53,16 +53,6 @@ def _update_dbt_group(
             return
 
         artifact_full_path_split = artifact.normalized_full_path.split(posixpath.sep)
-        if isinstance(artifact, NormalizedExposureSchema):
-            # For exposures, we want the path to be the path in the BI, but to start with <package name>/<artifact type>
-            # like other artifacts.
-            # NOTE - if there is no path provided in the BI, the FQN will just be the exposure name.
-            artifact_full_path_split = []
-            if artifact.package_name:
-                artifact_full_path_split.append(artifact.package_name)
-            artifact_full_path_split.append("exposures")
-            artifact_full_path_split.extend(artifact.fqn.split("/"))
```

Comment:
Removing from here, because below we're making `normalized_full_path` reflect in the first place this logic (that guarantees `normalized_full_path` and the sidebar are aligned, which the FE lineage filtering relies on)